src/main/java/com/github/hokkaydo/eplbot/module/code/CodeCommand.java

```diff
@@ -0,0 +1,220 @@
+        if(event.getInteraction().getType() != InteractionType.MODAL_SUBMIT || !event.getModalId().contains("Code")) return;
+        Optional<ModalMapping> lang = Optional.ofNullable(event.getInteraction().getValue("language"));
+        Optional<ModalMapping> body = Optional.ofNullable(event.getInteraction().getValue("body"));
+        Guild guild = event.getGuild();
+        if(lang.isEmpty() || body.isEmpty() || guild == null) return;
+
+        Integer runTimeout = Config.getGuildVariable(guild.getIdLong(), "COMMAND_CODE_TIMELIMIT");
+        event.getInteraction().reply("Processing since: <t:" + Instant.now().getEpochSecond() + ":R>").queue();
+
+        String languageOption = Objects.requireNonNull(lang.get().getAsString());
+        String bodyStr = Objects.requireNonNull(body.get().getAsString());
+        Runner runner = RUNNER_MAP.get(languageOption);
+        performSubmit(event.getMessageChannel(),bodyStr, languageOption);
+        performResponse(event.getMessageChannel(),runner.run(bodyStr, runTimeout));
+    }
+    private void performSubmit(MessageChannel textChannel, String bodyStr,String codeName){
+        if (!messageLengthCheck(bodyStr)){
+            sendSubmittedAsText(textChannel,bodyStr,codeName);
+            return;
+        }
+        File submitted = createSubmittedAsFile(bodyStr);
+        if (!exceed8mbOfData(submitted)){
+            if (submitted != null){
+                sendSubmittedAsFile(textChannel,submitted);
+                return;
+            }
+            sendErrorMessage(textChannel,Strings.getString("COMMAND_CODE_COULDNT_WRITE_THE_FILE"));
+            return;
+        }            
+        sendErrorMessage(textChannel,Strings.getString("COMMAND_CODE_EXCEEDEDFILESIZE"));
+    }
+    private void performResponse(MessageChannel textChannel, String result){
+        if (!messageLengthCheck(result)){
+            sendResponseAsText(textChannel,result);
+            return;
+        }
+        File submitted = createResponseAsFile(result);
+        if (!exceed8mbOfData(submitted)){
+            
+            if (submitted != null){
+                sendResponseAsFile(textChannel,submitted);
+                return;
+            }
+            sendErrorMessage(textChannel,Strings.getString("COMMAND_CODE_COULDNT_WRITE_THE_FILE"));
+            return;
+        }            
+        sendErrorMessage(textChannel,Strings.getString("COMMAND_CODE_EXCEEDEDFILESIZE"));
+    }
+    private void sendErrorMessage(MessageChannel textChannel, String content){
+        textChannel.sendMessage(content).queue();
+    }
+    private void sendSubmittedAsText(MessageChannel textChannel, String bodyStr, String codeName){
+        textChannel.sendMessage("```"+codeName.toLowerCase()+"\n"+bodyStr+"\n```").queue();
+    }
+    private void sendResponseAsText(MessageChannel textChannel, String result){
+        textChannel.sendMessage("`"+result+"`").queue();
+    }
+    private boolean messageLengthCheck(String content){
```

Comment:
Rename to "validateMessageLength" and invert the condition in the code (you always use  `!messageLengthCheck(..)`)